kineticAR.md

```diff
@@ -0,0 +1,285 @@
+# KineticAR interfaces
+
+This is a reference to build a [**Vuforia Spatial Toolbox** (VST)](https://forum.spatialtoolbox.vuforia.com ) KineticAR prototype using robotic interfaces for any moving robot in 2D or 3D dimensions. 
```

Comment:
The end should either be "2D or 3D" or "2 or 3 dimensions" since it's a bit redundant